### PR TITLE
chore: update local debug wordings

### PR DESCRIPTION
**Pull request by swatDong**
- adjust comment for applying new tasks to old project
- adjust comment for botMessagingEndpoint
- adjust FAQ doc

### CODE REVIEW

**Review by XiaofuHuang — DISMISSED** [review]

**Review by kuojianlu — APPROVED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts [hunk 1]
```diff
@@ -17,7 +17,8 @@ export function generateTasksJson(
   const comment = `
   // This file is automatically generated by Teams Toolkit.
   // The teamsfx tasks defined in this file require Teams Toolkit version >= 4.0.7.
-  // See https://aka.ms/teamsfx-debug-tasks to know the details and how to customize each task.
+  // See https://aka.ms/teamsfx-debug-tasks to know the details and how to customize each task,
```
Comment (jidddddd) [inline packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts:20]: See https://aka.ms/teamsfx-debug-tasks for details on how to customize each task and how to integrate with existing Teams Toolkit projects.

File: packages/fx-core/tests/plugins/solution/debug/util/tasksTransparency.test.ts [hunk 2]
```diff
@@ -286,7 +286,8 @@ describe("tasksTransparency", () => {
 
 const expected = `// This file is automatically generated by Teams Toolkit.
 // The teamsfx tasks defined in this file require Teams Toolkit version >= 4.0.7.
-// See https://aka.ms/teamsfx-debug-tasks to know the details and how to customize each task.
+// See https://aka.ms/teamsfx-debug-tasks to know the details and how to customize each task,
```
Comment (jidddddd) [inline packages/fx-core/tests/plugins/solution/debug/util/tasksTransparency.test.ts:289]: See https://aka.ms/teamsfx-debug-tasks for details on how to customize each task and how to integrate with existing Teams Toolkit projects.

File: packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts (hunk 1 above)
Comment (swatDong) [reply packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksTransparency.ts]: Done

File: packages/fx-core/tests/plugins/solution/debug/util/tasksTransparency.test.ts (hunk 2 above)
Comment (swatDong) [reply packages/fx-core/tests/plugins/solution/debug/util/tasksTransparency.test.ts]: Done

**Review by jidddddd — APPROVED** [review]

**Review by kuojianlu — DISMISSED** [review]

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**